### ISSUE TITLE
Add JavaDocs Documentation

### ISSUE THREAD

**Issue by adamfisher** [martijn00/ExoPlayerXamarin#7]
Request to add JavaDocs in the bindings for improved documentation and parameter names that more closely matches the native library.
- https://developer.xamarin.com/guides/android/advanced_topics/java_integration_overview/binding_a_java_library_(.jar)/naming_parameters_with_javadoc/
- https://forums.xamarin.com/discussion/11121/including-javadocs-in-a-binding-library

**Comment by adamfisher** [comment]
Found the [JavaDocs jar for r1.4.2](https://bintray.com/google/exoplayer/exoplayer/view#files) but having issues integrating it. I tried unzipping the docs and placing them in a folder called "JavaDocs" in the same folder as the binding project. I also tried `Java7DocPaths` as recommended in that forum thread but the documentation/parameter names on methods are not getting updated.

```xml
<JavaDocPaths>JavaDocs</JavaDocPaths>
```
